conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ServiceExceptionMapper.java

```diff
@@ -55,23 +54,13 @@ public Response toResponseInner(ServiceException exception) {
                     exception);
         }
 
-        ResponseBuilder builder = Response.status(httpStatus);
-        try {
-            SerializableError error = SerializableError.builder()
-                    .from(SerializableError.forException(exception))
-                    .build();
-            builder.type(MediaType.APPLICATION_JSON);
-            builder.entity(error);
-        } catch (RuntimeException e) {
-            log.warn(
-                    "Unable to translate exception to json",
-                    SafeArg.of("errorInstanceId", exception.getErrorInstanceId()),
-                    e);
-            // simply write out the exception message
-            builder.type(MediaType.TEXT_PLAIN);
-            builder.entity("Unable to translate exception to json. Refer to the server logs with this errorInstanceId: "
-                    + exception.getErrorInstanceId());
-        }
-        return builder.build();
+        SerializableError error = SerializableError.builder()
+                .from(SerializableError.forException(exception))
+                .build();
```

Comment:
I realize this was the same before, but is there any reason for the additional builder?
```suggestion
        SerializableError error = SerializableError.forException(exception);
```

Reply:
No, will remove